editor/plugins/sprite_frames_editor_plugin.h

```diff
@@ -39,25 +39,25 @@
 #include "scene/gui/split_container.h"
 #include "scene/gui/tree.h"
 
-class SpriteFramesEditor : public PanelContainer {
```

Comment:
I'm getting the following error on latest master:

```
ERROR: Class SpriteFramesEditor already has a method _gui_input
   At: core\class_db.cpp:1216
```

This was very likely caused by changing the base class to `HSplitContainer` here.

I think removing this line would be all that's needed to fix it: https://github.com/godotengine/godot/blob/e190589f3debad7bb20f62288d13f99378a0dbe2/editor/plugins/sprite_frames_editor_plugin.cpp#L645

This is because `SplitContainer` already register this method: https://github.com/godotengine/godot/blob/e190589f3debad7bb20f62288d13f99378a0dbe2/scene/gui/split_container.cpp#L325